torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -888,9 +888,22 @@ Operator(                                                                      \
     DEFINE_INT_FLOAT_OP(aten::remainder, fmod((b + fmod(a, b)), b), float)
 
 
-    // TODO: Support python floordiv (//)
-    // Right now aten::floordiv is only used by loop unrolling
-    DEFINE_INT_OP(aten::floordiv, a / b)
+    // in c++ int division rounds to the integer closer to 0, in python floordiv
+    // rounds to lower integer
+    DEFINE_INT_OP(aten::floordiv, int64_t(std::floor(float(a) / float(b))))
```

Comment:
I think we can combine the DEFINE_INT_OP here and the following `aten::floordiv(float a, float b) -> float` into a single DEFINE_GENERIC_OP, see aten::reminder as a example